src/app/containers/Ad/Canonical/index.test.jsx

```diff
@@ -25,3 +26,37 @@ describe('CanonicalAds Ads', () => {
     );
   });
 });
+
+jest.mock('#lib/utilities/isLive', () => jest.fn());
+
+describe('getBootsrapSrc', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  [
+    {
+      queryString: '?ads-js-env=live',
+      isLiveValue: false,
+      bootstrapSrc:
+        'https://gn-web-assets.api.bbc.com/ngas/dotcom-bootstrap.js',
+    },
+    {
+      queryString: '?khoaphan',
```

Comment:
Can we use another query? 😄
```suggestion
      queryString: '?khoaphan',
```

Reply:
Forgot I put that there 😆 👀